website/source/docs/enterprise/network-segments/index.html.md

```diff
@@ -18,6 +18,8 @@ with each other.
 To get started with Network Segments,
 [read the guide](https://learn.hashicorp.com/consul/day-2-operations/network-segments).
 
+-> **Note:** Due to limitations in [Serf](https://www.consul.io/docs/internals/gossip.html), you may currently only use up to 10 Network Segments per Consul cluster.
```

Comment:
```suggestion
-> **Note:** Due to limitations in [Serf](https://www.consul.io/docs/internals/gossip.html), a Consul agent configured with too many Network Segments may not be able to start```

Reply:
We should probably remove the arbitrary value of 10 Network Segments.

Reply:
+1 if 10 isn't a real number we shouldn't say it, or we should hedge with "as few as" or something?